Tests/CocoaLumberjackTests/DDContextFilterLogFormatterTests.m

```diff
@@ -0,0 +1,176 @@
+    XCTAssertNil([self.filterLogFormatter formatLogMessage:testLogMessage()]);
+    
+    [self.filterLogFormatter addToWhitelist:1];
+    XCTAssertEqualObjects([self.filterLogFormatter whitelist], @[@1]);
+    XCTAssertTrue([self.filterLogFormatter isOnWhitelist:1]);
+    XCTAssertEqualObjects([self.filterLogFormatter formatLogMessage:testLogMessage()], @"test log message");
+    
+    [self.filterLogFormatter removeFromWhitelist:1];
+    XCTAssertEqualObjects([self.filterLogFormatter whitelist], @[]);
+    XCTAssertFalse([self.filterLogFormatter isOnWhitelist:1]);
+    XCTAssertNil([self.filterLogFormatter formatLogMessage:testLogMessage()]);
+}
+
+@end
+#pragma clang diagnostic pop
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma mark -
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+@interface DDContextDenylistFilterLogFormatterTests : XCTestCase
+@property (nonatomic, strong, readwrite) DDContextDenylistFilterLogFormatter *filterLogFormatter;
+@end
+
+@implementation DDContextDenylistFilterLogFormatterTests
+
+- (void)setUp {
+    [super setUp];
+    self.filterLogFormatter = [[DDContextDenylistFilterLogFormatter alloc] init];
+}
+
+- (void)tearDown {
+    self.filterLogFormatter = nil;
+    [super tearDown];
+}
+
+- (void)testDDContextDenylistFilterLogFormatterTests {
+    XCTAssertEqualObjects([self.filterLogFormatter denylist], @[]);
+    XCTAssertFalse([self.filterLogFormatter isOnDenylist:1]);
+    XCTAssertEqualObjects([self.filterLogFormatter formatLogMessage:testLogMessage()], @"test log message");
+    
+    [self.filterLogFormatter addToDenylist:1];
+    XCTAssertEqualObjects([self.filterLogFormatter denylist], @[@1]);
+    XCTAssertTrue([self.filterLogFormatter isOnDenylist:1]);
+    XCTAssertNil([self.filterLogFormatter formatLogMessage:testLogMessage()]);
+    
+    [self.filterLogFormatter removeFromDenylist:1];
+    XCTAssertEqualObjects([self.filterLogFormatter denylist], @[]);
+    XCTAssertFalse([self.filterLogFormatter isOnDenylist:1]);
+    XCTAssertEqualObjects([self.filterLogFormatter formatLogMessage:testLogMessage()], @"test log message");
+}
+
+@end
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+@interface DDContextBlacklistFilterLogFormatterTests : XCTestCase
+@property (nonatomic, strong, readwrite) DDContextBlacklistFilterLogFormatter *filterLogFormatter;
+@end
+
+@implementation DDContextBlacklistFilterLogFormatterTests
+
+- (void)setUp {
+    [super setUp];
+    self.filterLogFormatter = [[DDContextBlacklistFilterLogFormatter alloc] init];
+}
+
+- (void)tearDown {
+    self.filterLogFormatter = nil;
+    [super tearDown];
+}
+
+- (void)testDDContextBlacklistFilterLogFormatterTests {
+    XCTAssertEqualObjects([self.filterLogFormatter blacklist], @[]);
```

Comment:
Move this method to `DDContextDenylistFilterLogFormatterTests` and add deprecated attribute.
Also add comment "for compatibility".